classes/config_helper.php

```diff
@@ -0,0 +1,154 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace tool_objectfs;
+
+use stdClass;
+
+defined('MOODLE_INTERNAL') || die();
+
+define('OBJECT_LOCATION_ERROR', -1);
+define('OBJECT_LOCATION_LOCAL', 0);
+define('OBJECT_LOCATION_DUPLICATED', 1);
+define('OBJECT_LOCATION_EXTERNAL', 2);
+
+define('OBJECTFS_REPORT_OBJECT_LOCATION', 0);
+define('OBJECTFS_REPORT_LOG_SIZE', 1);
+define('OBJECTFS_REPORT_MIME_TYPE', 2);
+
+define('OBJECTFS_BYTES_IN_TERABYTE', 1099511627776);
+
+class config_helper {
```

Comment:
give that this class contains a few things which are unrelated to config, how about calling this something more generic like ' tool_objectfs\local\manager' ?